test/cli/template-use/complex/in/repository/my-template/v1/src/other/keboola.orchestrator/orchestrator/phases/002-phase-2/phase.jsonnet

```diff
@@ -1,6 +1,6 @@
 {
   name: "Phase 2",
   dependsOn: [
-    "001-phase-1",
+    "<previous>",
   ],
```

Comment:
A tu je 2. `phase`, kde sa `<previous>` aplikuje.
Funguje to rovnako ako pred zmenou, preto sa nic viac v teste nemeni.